### ISSUE TITLE
After upgrade to 9.10.1 Error SecurityController.UpdateOtherSettings

### ISSUE THREAD

**Issue by sboshuis** [dnnsoftware/Dnn.Platform#4839]
## Description of bug
After upgrade to 9.10.1 Error SecurityController.UpdateOtherSettings

ExceptionType:System.NullReferenceException,StackTrace:   bij DotNetNuke.Common.Utilities.Config.SetMaxUploadSize(Int64 newSize)

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to Security settings more settings where e.g. max file upload size is
2. Open Console in F12
3. Change something and Save
4. Error 500 in network tab on UpdateOtherSettings

## Expected behavior
No Error 500

## Error information
I would propose to add some checking in the code so if the attribute is missing then add it to the node
in the public static void SetMaxUploadSize(long newSize)

![image](https://user-images.githubusercontent.com/37078656/134353083-c82acc3f-f159-40bc-b25b-378955252b16.png)

Basically in the web config it looks like

![image](https://user-images.githubusercontent.com/37078656/134354444-342b371a-0803-46fc-b47a-5b8185de39ff.png)

(please note that often the attribute maxAllowedContentLength is not there and just falls back to its default)
So the code wants to set an attribute that is not there and should be added in this case.

## Affected version

* [ ] 10.00.00 alpha build
* [X] 09.10.01 latest supported release

**Comment by sboshuis** [comment]
work around for anyone facing this issue like me is manually adding maxAllowedContentLength with a value to requestLimits node in security/requestfiltering than go back to dnn and press save again. Now it will save without error

**Comment by david-poindexter** [comment]
Thanks for logging this issue @sboshuis - what version of DNN were you upgrading from?  I am mot sure if this is related to the issue or not, but just in case, it may be helpful to know.

**Comment by sboshuis** [comment]
in this case 9.8.1  but I have many more web configs where this tag is not there

**Comment by hismightiness** [comment]
I don't see this in the web.config in any of the sites I've checked so far.

**Comment by david-poindexter** [comment]
Thanks for confirming @hismightiness 🎉

**Comment by valadas** [comment]
I cannot make this happen in 9.10.1 and also don't have that node in any of the web.config files I can see on many past versions. I am closing this issue but if you can come up with accurate steps for a developer to make this happen, please comment and I'll be happy to reopen.

**Comment by sboshuis** [comment]
Will give you extra info how to replicate I can do this in any of my installs and retested just now and made print screens for you. will contact you on slack

**Comment by valadas** [comment]
In my case, that value changes the maxRequestLenght on the configuration->httpRuntime value and I have no security node in the web.config

**Comment by hismightiness** [comment]
@sboshuis Basically, we need to know how to duplicate something before anyone can realistically know what/how to fix the issue.  In this case, everyone in this thread manages many DNN sites and none of us see this behavior yet.  If you can provide the consistent steps to reproduce the issue, we'd be able to troubleshoot this further together.
